Kitodo-LongTimePreservationValidationModule/src/main/java/org/kitodo/longtimepreservationvalidationmodule/LongTimePreservationValidationModule.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.longtimepreservationvalidationmodule;
+
+import edu.harvard.hul.ois.jhove.App;
+import edu.harvard.hul.ois.jhove.JhoveBase;
+
+import java.net.URI;
+
+import org.kitodo.api.validation.ValidationResult;
+import org.kitodo.api.validation.longtimepreservation.FileType;
+import org.kitodo.api.validation.longtimepreservation.LongTimePreservationValidationInterface;
+import org.kitodo.config.Config;
+
+/**
+ * An LongTimePreservationValidationInterface implementation using Jhove.
+ */
+public class LongTimePreservationValidationModule implements LongTimePreservationValidationInterface {
+    /**
+     * Path to the {@code jhove.conf} file
+     *
+     * <p>
+     * Example:
+     * LongTimePreservationValidationModule.jhoveConf=/var/lib/tomcat8/webapps/kitodo-production/WEB-INF/classes/jhove.conf
+     */
+    private static final String PARAMETER_JHOVE_CONF = "LongTimePreservationValidationModule.jhoveConf";
```

Comment:
This must be adjusted as pull request #1623 is merged. Please add this to your todo list.